server/moodle/mod/livequiz/classes/livequiz/livequiz.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace mod_livequiz\livequiz;
+
+use DateTime;
+use dml_exception;
+use mod_livequiz\answers\answers;
+use mod_livequiz\question\question;
+use mod_livequiz\questions_answers_relation\questions_answers_relation;
+use mod_livequiz\quiz_questions_relation\quiz_questions_relation;
+use stdClass;
+
+/**
+ * This class is essentially supposed to be static.
```

Comment:
Why is this essentially a static class? Will we never work with a livequiz object?